worker-workflow/src/main/resources/workflow-control.js

```diff
@@ -184,6 +184,16 @@ function fieldExists(document, fieldName) {
     return document.getField(fieldName).hasValues();
 }
 
+function isNotEmptyMap(obj) {   
+    if(!obj)
+        return false;
+    var jsonObj = JSON.parse(obj);
+    if (Object.getOwnPropertyNames(jsonObj).length === 0) {
```

Comment:
simplfy to return !(Object.getOwnPropertyNames(jsonObj).length === 0);

Reply:
(or even to `return !!Object.getOwnPropertyNames(jsonObj).length;`)